internal/jobs/launch_instance_aws.go

```diff
@@ -42,8 +42,13 @@ type LaunchInstanceAWSTaskArgs struct {
 	ARN *clients.Authentication
 }
 
-// Unmarshall arguments and handle error
+// HandleLaunchInstanceAWS unmarshalls arguments and handles error
 func HandleLaunchInstanceAWS(ctx context.Context, job *worker.Job) {
+	if job == nil {
+		zerolog.Ctx(ctx).Error().Msg("No job for HandleLaunchInstanceAWS")
```

Comment:
nit: If you prefer it, maybe we should add a logger? 
```suggestion
	logger := zerolog.Ctx(ctx)
	logger.Debug().Msg("Started instance launch AWS job")
	if job == nil {
			logger.Debug().Msg("No job for HandleLaunchInstanceAWS")
		}
```

Reply:
ditto on other providers